src/hcf-release/src/acceptance-tests-brain/test-resources/node-env/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "node-env",
   "version": "1.0.0",
   "engines": {
-    "node": "5.x"
   },
```

Comment:
I don't really care either way; I've now removed `engines` completely and force-pushed.

Or do you want to remove `dependencies` as well?

```
  "dependencies": {}
```